### DISCUSSION TITLE
glb/gltf model texture visualization

### DISCUSSION THREAD

**Discussion by boweih** [isl-org/Open3D#4538] [Q&A] [answered]
Hi all,

I have some .glb format models with texture and want to visualize them, while it alarm me that "[Open3D WARNING] Read geometry::Image failed: missing file extension." when I call read_triangle_mesh(). I wonder if has anything do with the embedded binary texture instead of external .png format texture?

**Comment by errissa** [comment, marked as answer]
That is correct. Open3D doesn't currently support extracting the textures from the glb.

**Reply by boweih** [reply]
Thank you for the answer! It would be great to put it in the document as it took me whole day trying to fix :(
